interpreter/meta/jslib/wast.ml

```diff
@@ -0,0 +1,25 @@
+open Js_of_ocaml
```

Comment:
If you introduce this as wast.ml, shouldn't you remove the prior wasm.ml?

Reply:
I've now removed `wasm.ml` and associated build files in the latest commits.